module_utils/network_lsr/argument_validator.py

```diff
@@ -465,6 +475,22 @@ def _validate_post(self, value, name, result):
 
 
 class ArgValidator_DictIP(ArgValidatorDict):
+    VALID_MODES = [
```

Comment:
this variable doesn't seem to have a good name.

Reply:
variable name VALID_MODES is widely  used in the file, I just want to keep it consistent. But maybe you are right. Updated!